runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -143,6 +143,9 @@ private void register() {
 
     /**
      * Adds a destination to LogReplicationModelMetadataTable for monitoring.
+     * <p>
+     * A new logical group will be created if one does not already exist.
+     * The destination will not be added again if it already exists in the group.
```

Comment:
Should we add that an exception is thrown if either group or destination is empty or null?